src/app/modules/practitioner-registration/pages/facility-info/facility-info.component.ts

```diff
@@ -137,7 +135,6 @@ export class FacilityInfoComponent extends BcpBaseForm implements OnInit, AfterV
           }
 
         }, error => {
-          console.log('ARC apiService onerror', error);
```

Comment:
We may want to keep these console.logs just in case we fall into these cases.

Reply:
Added these back.